python/cvmfs/dirent.py

```diff
@@ -131,11 +130,11 @@ def _read_content_hash_type(self):
         bit_mask     = _Flags.ContentHashType
         right_shifts = 0
         while bit_mask & 1 == 0:
-            bit_mask = bit_mask >> 1
+            bit_mask >>= 1
             right_shifts += 1
         hash_type = ((self.flags & _Flags.ContentHashType) >> right_shifts) + 1
         self.content_hash_type = \
-            hash_type if hash_type > 0 and hash_type < ContentHashTypes.UpperBound \
+            hash_type if 0 < hash_type < ContentHashTypes.UpperBound \
```

Comment:
![O Rly?](http://www.jfrog.com/wp-content/uploads/2014/06/O-rly001.jpg)

You can specify ranges like that in Python?